app/sidebar.py

```diff
@@ -138,20 +144,25 @@ def make_sidebar(api: PtxboaAPI):
     )
     st.session_state["transport"] = st.sidebar.radio(
         "Mode of transportation (for selected supply country):",
-        api.get_dimension("transport").index,
+        ["Ship", "Pipeline"],

```

Comment:
@markushal adressed your requested changes. I also removed the `api.get_dimensions` call in those two instances. Creating a list of two entries takes 27 ns, creating a pd.DataFrame with two entries and accessing its index takes125 µs